app.py

```diff
@@ -58,6 +82,10 @@ def sync(dst: str, debug: bool = False, force: bool = False) -> None:
 
     _LOGGER.info("Running document syncing job in version %r", __component_version__)
 
+    number_synced_documents = 0
+    number_unsynced_documents = 0
+    number_skypped_documents = 0
```

Comment:
Might be nitpicking :)

```suggestion
    number_skipped_documents = 0
```

Reply:
I like this :D thanks @fridex !! 🍻